core/src/main/java/org/mskcc/cbio/portal/model/MutationalSignature.java

```diff
@@ -0,0 +1,46 @@
+package org.mskcc.cbio.portal.model;
```

Comment:
We should rename this class MutationalSignatureMeta to avoid clash with existing MutationalSignature class.